starter-code/css/main.css

```diff
@@ -5,48 +5,174 @@ Text: 1A1A1A
 Green: #00B172
 White: #FFF
 
-*/
+ */
+ @import url('https://fonts.googleapis.com/css?family=Roboto:100,300,400,500,700');
 
 body {
     margin: 0 auto;
+    font-family: "roboto";
 }
 
 .header {
-    height: 100px;
+    height: 60px;
     width: 100%;
 }
 
 .green-logo {
    height: 50px;
    float: left;
+   margin: 20px 0 0 20px;
 }
 
 .menu-nav {
     list-style-type: none;
     text-align: center;
-    margin: 0 auto;
+    margin: 0 20px;
```

Comment:
You can get rid of the margin and only use the padding and the flexbox 
   display : flex;
    justify-content: space-between;
and 
  padding: 30px 50px 0 750px;